Create ui-c8y-1020-16-0-handle-projects-created-with-standalone-flag-set-to-true-wh-6414.md (#2351)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-16-0-handle-projects-created-with-standalone-flag-set-to-true-wh-6414.md

---
date: ""

> title: Handle projects created with standalone flag set to true wh… (#6414) [GRAFT][release/cd] (#6805)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Handle projects created with standalone flag set to true when importing
```

> title: Handle newly created Angular projects created with standalone flag set to true, when adding Cumulocity IoT schematics

Reviewer: ```suggestion
title: Cumulocity IoT schematics can successfully be integrated into Angular standalone projects
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-56831
version: 1020.16.0
---

> Handle projects created with standalone flag set to true wh… (#6414) [GRAFT][release/cd] (#6805)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, when importing a project that was originally created with the standalone flag set to true, the import process would fail. This has now been fixed so that such projects can be successfully imported. This change improves the user experience by allowing standalone projects to be imported without any errors, ensuring a smooth migration process for users who want to move their standalone projects into a different environment.
```

> Previously, adding {{< product-c8y-iot >}} schematics to Angular projects initialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.

Reviewer: ```suggestion
Previously, adding {{< product-c8y-iot >}} schematics to Angular projects initialized with the standalone flag set to true failed. This issue has been addressed. Now, when integrating {{< product-c8y-iot >}} schematics into standalone Angular projects, the standalone flag is automatically set to false. Full support for standalone projects will be implemented in future updates. This interim solution allows developers to use {{< product-c8y-iot >}} schematics with projects initially set up as standalone, though with standalone features disabled.

@BeateRixen
Member
[BeateRixen](https://github.com/BeateRixen) commented [3 days ago](https://github.com/SoftwareAG/c8y-docs/pull/2351#issuecomment-2277578967)
Sounds good to me. What do you think @rahultsag?

@rahultsag
Collaborator
[rahultsag](https://github.com/rahultsag) commented [17 minutes ago](https://github.com/SoftwareAG/c8y-docs/pull/2351#issuecomment-2283488968)
Yes,text proposed by Dawid provide more accurate info to developers reading the changelog. Please use that.

@BeateRixen
[Update content/change-logs/application-enablement/ui-c8y-1020-16-0-ha…](https://github.com/SoftwareAG/c8y-docs/pull/2351/commits/68a5934d22ab0bbfb1ac0d3ee8290632c0d73fc1) 
68a5934
Merge state
This branch has not been deployed
No deployments
Code owner review required
Waiting on code owner review from BeateRixen, MWindel, janhommes, and/or rahultsag. 
4 pending reviewers
View
All conversations are resolved
6 resolved conversations
Some checks haven’t completed yet
1 queued check
@github-actions
CI / Build Hugo (pull_request) Queued — Waiting to run this check...
Required
[Details](https://github.com/SoftwareAG/c8y-docs/actions/runs/10349535806/job/28644085276?pr=2351)
This branch is out-of-date with the base branch
Merge the latest changes from develop into this branch.
Merging is blocked
Merging can be performed automatically with 2 approving reviews.
 Merge without waiting for requirements to be met (bypass branch protections)

 Automatically merge when all requirements are met. Learn more
@BeateRixen

Add a comment
Comment
 
Add your comment here...
 
Remember, contributions to this repository should follow our GitHub Community Guidelines.
 ProTip! Add comments to specific lines under Files changed.
Reviewers
@BeateRixen
[BeateRixen](https://github.com/BeateRixen)
@[rahultsag](https://github.com/rahultsag)
rahultsag
@[janhommes](https://github.com/janhommes)
janhommes
@[MWindel](https://github.com/MWindel)
MWindel
@[DawidSAG](https://github.com/DawidSAG)
DawidSAG
At least 2 approving reviews are required to merge this pull request.

Still in progress?
Assignees
No one—
Labels
automated-change-note
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet

Notifications
Customize
You’re receiving notifications because you were mentioned.
4 participants
@gen-App-Enablement
@DawidSAG
@BeateRixen
@rahultsag
```

> Previously, adding {{< product-c8y-iot >}} schematics to Angular projects initialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.

Reviewer: ```suggestion
Previously, adding {{< product-c8y-iot >}} schematics to Angular projects initialized with the standalone flag set to true failed. This issue has been addressed. Now, when integrating {{< product-c8y-iot >}} schematics into standalone Angular projects, the standalone flag is automatically set to false. Full support for standalone projects will be implemented in future updates. This interim solution allows developers to use {{< product-c8y-iot >}} schematics with projects initially set up as standalone, though with standalone features disabled.
```

> Previously, adding {{< product-c8y-iot >}} schematics to Angular standalone projectsinitialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.

Reviewer: ```suggestion
Previously, adding {{< product-c8y-iot >}} schematics to Angular standalone projects initialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.
```

> Previously, adding {{< product-c8y-iot >}} schematics to Angular standalone projects initialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.

Reviewer: ```suggestion
Previously, adding {{< product-c8y-iot >}} schematics to Angular projects initialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.
```

ticket: MTM-56831
version: 1020.16.0
---

> Previously, adding Cumulocity IoT schematics to Angular projects initialized with the standalone flag set to true would fail. This issue has been resolved, allowing successful integration of Cumulocity IoT schematics into standalone Angular projects.

Reviewer: ```suggestion
Previously, adding {{< product-c8y-iot >}} schematics to Angular standalone projectsinitialized with the standalone flag set to true failed. This issue has been resolved, allowing successful integration of {{< product-c8y-iot >}}schematics into standalone Angular projects.
```
